kalite/django_cherrypy_wsgiserver/cherrypyserver.py

```diff
@@ -30,6 +30,14 @@ def start(self):
 
         # Serve the content files
         if getattr(settings, "CONTENT_ROOT", None):
+            # Assessment items
+            from kalite.contentload.settings import KHAN_ASSESSMENT_ITEM_ROOT
+            static_handler = cherrypy.tools.staticdir.handler(
+                section="/",
+                dir="",
+                root=os.path.abspath(KHAN_ASSESSMENT_ITEM_ROOT)
+            )
+            cherrypy.tree.mount(static_handler, settings.CONTENT_URL + "khan/")
```

Comment:
KHAN_ASSESSMENT_ITEM_ROOT can be changed, so we have to serve it as it was configured rather than expecting it to always be nested in CONTENT_ROOT